tests/generic_config_updater/generic_updater_test.py

```diff
@@ -12,6 +12,8 @@
 # sys.path.insert(0,'../../generic_config_updater')
 # import generic_updater as gu
 
+init_tests()
```

Comment:
>init_tests [](http://example.com/codeflow?start=0&length=10)

Normally we don't write code outside the test class/functions in file scope. Do you want to use a fixture [https://docs.pytest.org/en/latest/how-to/fixtures.html](https://docs.pytest.org/en/latest/how-to/fixtures.html)?

I guess it fits into a session fixture.